<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DomenEditorWindow</class>
 <widget class="QMainWindow" name="DomenEditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>375</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>351</width>
    <height>375</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>351</width>
    <height>375</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Редактирование домена</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="domenNameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя домена:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="domenNameText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="domenValLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Значения:</string>
    </property>
   </widget>
   <widget class="QListView" name="domenValView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>331</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="domenValText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="domenValInpLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Значение:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="domenAddButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>229</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Добавить</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delDomenValButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>331</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>331</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Отмена</string>
    </property>
   </widget>
   <widget class="QPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
